### PR TITLE
Fix/handle blue square permissions

### PR DESCRIPTION
**Pull request by guiwustro**

### CODE REVIEW

**Comment by guiwustro** [comment]
This PR is fixing the bug enabling a permission handleSquare

This PR is adding a props to a component that was forgotten, that enables a person with the permission of handleBlueSquares be able to delete/add BlueSquares.

To test this you should:
1. Login as administrator ---> Other Links --> Permissions Management --> Manage User Permissions ---> Select a user volunteer account that u have  and give "Only Assign Blue Squares" and "Handle Blue Squares". 
2. Login the volunteer account that received the permissions --> open a userprofile --> check if the Blue Squares "+" is enable---> http://localhost:3000/userprofile/63a4afbe3bdd831f7431320a

P.S. I tried like 3 times remove the ESLint stuff that was made in another file, sorry.

**Review by rauleffting — APPROVED** [review]
You simply added a property to the component BlueSquareLayout.jsx. Everything looks good. I also tested and worked fine. Thank you!

Screen Shot 2023-01-09 at 12 34 42

File: src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ import 'react-day-picker/lib/style.css';
 import dateFnsFormat from 'date-fns/format';
 import { Editor } from '@tinymce/tinymce-react';
 
-const AddTaskModal = (props) => {
+const AddTaskModal = props => {
```
Comment (EvianTan) [inline src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx:13]: No functional changes in this file.

Comment (guiwustro) [reply src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx:13]: Yess,I tried like 3 times remove the ESLint stuff that was made in this file, sorry.

Comment (EvianTan) [reply src/components/Projects/WBS/WBSDetail/AddTask/AddTaskModal.jsx:13]: No worries!

**Review by EvianTan — APPROVED** [review]

**Pull request merged**